### PR TITLE
hide rbf behind feature flag

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: packages/suite/src/config/suite/features.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@
 // General flags
 export const FLAGS = {
     GOOGLE_DRIVE_SYNC: false, // Google Drive sync (used for labeling)
+    RBF: false,
```
Comment (keraf) [inline packages/suite/src/config/suite/features.ts]: There should be a description of what the flag is about.

File: packages/suite/src/constants/wallet/sendForm.ts [hunk 2]
```diff
@@ -41,7 +41,8 @@ export const DEFAULT_OPRETURN = {
     dataHex: '',
 } as const;
 
-export const DEFAULT_OPTIONS = ['broadcast', 'bitcoinRBF'] as const;
+// export const DEFAULT_OPTIONS = ['broadcast', 'bitcoinRBF'] as const;
+export const DEFAULT_OPTIONS = ['broadcast'] as const;
```
Comment (keraf) [inline packages/suite/src/constants/wallet/sendForm.ts]: The goal of the feature flag is to flip a single switch to enable/disable a feature. This defeats the purpose of it as it requires code changes.

File: packages/suite/src/config/suite/features.ts (hunk 1 above)
Comment (mroz22) [reply packages/suite/src/config/suite/features.ts]: updated

File: packages/suite/src/constants/wallet/sendForm.ts (hunk 2 above)
Comment (mroz22) [reply packages/suite/src/constants/wallet/sendForm.ts]: true, updated

**Review by szymonlesisz — APPROVED** [review]

**Comment by szymonlesisz** [comment]
approving as a hotfix, however this flags/feature utils should be extended/reworked to be less painful to implement (got few ideas)

**Review by keraf — APPROVED** [review]

**Pull request merged**